### ISSUE TITLE
Feature Request - Support XChaCha20-Ploy1305

### ISSUE THREAD

**Issue by schlomie** [hardcore-sushi/DroidFS#52]
gocryptfs v2.2.0 was released [5 days ago](https://github.com/rfjakob/gocryptfs#v220-2021-09-25).

With it came the ability to use XChaCha20 instead of AES-GCM to boost performance on devices without AES hardware.

Having this as an option in DroidFS would be pretty terrific.

Related: If there is still some interest in supporting cryfs in DroidFS, cryfs from version [0.11.0](https://github.com/cryfs/cryfs/releases/tag/0.11.0) uses XChaCha20 as default as well.

**Comment by hardcore-sushi** [comment]
Yes I will upgrade the gocryptfs version. However I'm currently working on [#51](https://github.com/hardcore-sushi/DroidFS/issues/51) and I would like to finish this first (by the way maybe you have an idea that could save me 😅️)

**Comment by hardcore-sushi** [comment]
e47d9f4548ee6b96b94411872fd1499cb699530d: the feature will be available in the next release.
